.env.example

```
@@ -1,57 +1,88 @@
+# Hasura Configuration
+HASURA_URL=http://localhost:8080/v1/graphql            # Hasura GraphQL endpoint
+HASURA_GRAPHQL_ADMIN_SECRET=your_hasura_admin_secret   # Admin secret for Hasura
+CACHE_DB=your_cache_table_name                         # Cache table name
+RESPONSE_CACHE_DB=your_response_cache_table_name       # Response cache table name
+SEEKER_DB=your_seeker_table_name                       # Seeker table name
+ORDER_DB=your_order_table_name                         # Order table name
+TELEMETRY_DB=your_telemetry_table_name                 # Telemetry table name
+
+# Onest Protocol Configuration
+# Beckn domain
+DOMAIN=your_domain                     \
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)